tox.ini

```
@@ -59,6 +58,7 @@ extras =
 
 commands =
     devdeps: pip install -U --pre --only-binary :all: -i https://pypi.anaconda.org/scipy-wheels-nightly/simple numpy
+    devdeps: pip install -U -i https://pypi.anaconda.org/astropy/simple astropy --pre
```

Might as well? Or do you think it is not a good idea?

```suggestion
    devdeps: pip install -U --pre --only-binary :all: -i https://pypi.anaconda.org/scipy-wheels-nightly/simple scipy
    devdeps: pip install -U -i https://pypi.anaconda.org/astropy/simple astropy --pre
```

---

why not?